src/azure-cli/azure/cli/command_modules/appservice/commands.py

```diff
@@ -259,6 +259,10 @@ def load_command_table(self, _):
         g.custom_command('add', 'add_github_actions')
         g.custom_command('remove', 'remove_github_actions')
 
+    with self.command_group('functionapp deployment github-actions') as g:
```

Comment:
can we add some validation for this command to be not supported for other kinds?(functionapp only & not for LA, web apps).
Same changes to the webapp GitHub actions as well

Reply:
@haroonf there's some weird behavior here where if you run the command on a logic app, it runs successfully but the GitHub actions fails. We should definitely block the user from running the command if the app is a logicapp or webapp 

See here for how to check that the app kind is the right one: https://github.com/Azure/azure-cli/pull/21855/files#diff-335eb5f7ca32eba6e60e55af7ab1e404f19c4300681155fecaf3997f673d3739R395-R425

Reply:
added